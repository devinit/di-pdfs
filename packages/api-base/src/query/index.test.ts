import * as prettyFormat from 'pretty-format';
import {get} from '.';

describe('Test query', () => {
    it.skip('should query api', async () => {
        const query = `
            query China {
                overallMultiExpenditure {
                id
                year
                color
                value
                uid
                }
            }`;
        const data = await get('http://localhost:8080/graphql', query);
        expect(prettyFormat(data)).toMatchSnapshot();
     });
});
