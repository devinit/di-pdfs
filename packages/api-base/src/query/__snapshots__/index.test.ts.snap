// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test query should query api 1`] = `
"Object {
  \\"overallMultiExpenditure\\": Array [
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"Hy3y2Q5EM\\",
      \\"value\\": 0.2,
      \\"year\\": 2006,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"rkehyhQ9EM\\",
      \\"value\\": 0.2,
      \\"year\\": 2007,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"HyW2yn75EG\\",
      \\"value\\": 0.3,
      \\"year\\": 2008,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"rkMhy2Q5Ez\\",
      \\"value\\": 0.3,
      \\"year\\": 2009,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"Hkm2knXcVG\\",
      \\"value\\": 0.4,
      \\"year\\": 2010,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"BJN31nQqVM\\",
      \\"value\\": 0.3,
      \\"year\\": 2011,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"rkSn1n75Nz\\",
      \\"value\\": 0.4,
      \\"year\\": 2012,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"HJLn12Q9Nz\\",
      \\"value\\": 0.7,
      \\"year\\": 2013,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"HJDnJ2m9EG\\",
      \\"value\\": 0.4,
      \\"year\\": 2014,
    },
    Object {
      \\"color\\": \\"#e8443a\\",
      \\"id\\": \\"Multilateral organisations\\",
      \\"uid\\": \\"ry_3yn7qNM\\",
      \\"value\\": 0.6,
      \\"year\\": 2015,
    },
  ],
}"
`;
